### PR TITLE
add overview of AMY tables to documentation

### PR DESCRIPTION
**Pull request by maneesha**
This PR moves the documentation that was in the private systems repo to the AMY repo to make it more accessible to all AMY users.

### CODE REVIEW

**Comment by maneesha** [comment]
Not sure why this is asking for a migration

File: docs/amy_database_structure.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+## Events
+
+`workshops_event` - Primary table for all event data.
+
+### Commonly used fields
+
+* `id` Sequential, automatically assigned integer
+* `start` and `end` Event start and end dates 
+* `slug` Event's unique identifier, in the form YYYY-MM-DD-sitename
+* `url` Event's website. Typically in the format username.github.io/YYYY-MM-DD-sitename
```
Comment (pbanaszkiewicz) [inline docs/amy_database_structure.md:14]: Now we're supporting any sites, but these from GitHub make it easy to read event's slug.

File: docs/amy_database_structure.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+* `host_id` An integer representing the Event Host.  This is linked to the `workshops_organization` table
+* Location based fields:
+    * `venue` The venue name of the event
+    * `address` The street address of the event
+    * `latitude` and `longitude` Stored as floating point (decimal) numbers
+    * `country` Stored as the [two digit country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)
+* `contact` A pipe-separated (`|`) list of email addresses listed as contacts for that workshop
```
Comment (pbanaszkiewicz) [inline docs/amy_database_structure.md:21]: I'm pretty sure that the separator is `,` or `;` now.

File: docs/amy_database_structure.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+* `open_TTT_applications` Used only for instructor training events
+* `adminstrator_id` An integer representing the event organizer.  This is linked to the `workshops_organization` table. Historically any organization could be listed as an administrator. Recent updates to AMY limit this to Data Carpentry, Library Carpentry, Software Carpentry, The Carpentries, or self-organized. This enforcement is at the AMY app level, not at the database level.
+* `reg_key` Eventbrite registration key
+
+
+### Unused fields
+
+* `manual_attendance` We are no longer collecting or recording attendance
+* `admin_fee`, `invoice_status` We are not recording financial data in AMY
+* `repository_last_commit_hash`, `repository_metadata`, `metadata_all_changes`, `metadata_changed` Unknown
```
Comment (pbanaszkiewicz) [inline docs/amy_database_structure.md:34]: Related to reading workshop metadata changes, now (unfortunately) unused.

File: docs/amy_database_structure.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+* `instructors_post` `instructors_pre` `learners_longterm` `learners_post` `learners_pre` Previously used to store links to surveys.
+
+## Persons
+
+`workshops_persons` - Primary table for all person data. This includes all individuals, regardless of their role with The Carpentries.  
+
+### Commonly used fields
+
+* `id` Sequential, automatically assigned integer
+* `personal` `middle` `family` Three fields to hold the individual's name
```
Comment (pbanaszkiewicz) [inline docs/amy_database_structure.md:44]: It's worth adding that out of these 3, the only required is the `family`.

File: docs/amy_database_structure.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+* `email` Individual's primary email address. Used for user log in
+* `secondary_email` Alternate email address.
```
Comment (pbanaszkiewicz) [inline docs/amy_database_structure.md:46]: Optional.

**Comment by pbanaszkiewicz** [comment]
@maneesha feel free to merge or update with my comment. I wouldn't worry about the failed test, as long as all is well after the merge. (And if it's not, I'll come and fix it).

**Review by pbanaszkiewicz — APPROVED** [review]

**Pull request merged**